src/main/java/io/github/amerebagatelle/fabricskyboxes/skyboxes/textured/TexturedSkybox.java

```diff
@@ -55,7 +57,7 @@ public final void render(WorldRendererAccess worldRendererAccess, MatrixStack ma
         RenderSystem.enableBlend();
 
         blend.applyBlendFunc();
-        RenderSystem.setShader(GameRenderer::getPositionTexColorShader);
+        RenderSystem.setShaderColor(1.0F, 1.0F, 1.0F, 1.0F); // Todo: Calculate our brightness based off position of the player and biomes/height ranges/weather and move this into Blend.
```

Comment:
Why does this need changed/what might need rolled into Blend?  (just a "I've been away for too long" kinda question)

Reply:
Instead of changing the shader, we can just change the color of the current shader to blend(vanilla doesn't change the shader when rendering the skybox).

We might want this in Blend because it allows us to change brightness/color depending on the blend being used at the moment.(vanilla also does this but for the color the sky)